core/datastore/src/main/java/com/twix/datastore/AuthConfigure.kt

```
@@ -0,0 +1,52 @@
+
+internal object AuthConfigureSerializer : Serializer<AuthConfigure> {
+    override val defaultValue: AuthConfigure
+        get() = AuthConfigure()
+
+    override suspend fun readFrom(input: InputStream): AuthConfigure =
+        try {
+            Json.decodeFromString(
+                deserializer = AuthConfigure.serializer(),
+                string = input.readBytes().decodeToString(),
+            )
+        } catch (e: SerializationException) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)